mmv1/templates/terraform/examples/pubsub_topic_schema_settings.tf.erb

```diff
@@ -0,0 +1,8 @@
+resource "google_pubsub_topic" "<%= ctx[:primary_resource_id] %>" {
+  name = "<%= ctx[:vars]['topic_name'] %>"
+
+  schema_settings {
```

Comment:
Our CI is getting the following error with this example:
```
Error: Error creating Topic: googleapi: Error 400: Schema settings for the topic <topic-name-here> must include both the name and the encoding.
```

Reply:
Hi,
I included this change in the following PR: https://github.com/GoogleCloudPlatform/magic-modules/pull/4741 because for topic creation with schema in test the schema creation is needed as well. This PR can be closed